jbuilder.opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+version: "transition"
```

Comment:
CMIIW, shouldn't this be `transitions`? (or the one in `dune.opam` should be `transition`.)

Reply:
Indeed, it's the one in dune.opam that's wrong. Thanks for pointing that out